restaurant/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import "./App.css";
+import Home from "./assets/home";
+import Menu from "./assets/menu";
+import Contact from "./assets/contact";
+import { useState } from "react";
+function App() {
```

Comment:
Add an empty line after imports and before your first block of code for logical separation.